pkg/load/agents/configAgent.go

```diff
@@ -62,10 +62,9 @@ func NewFakeConfigAgent(configs load.ByOrgRepo) ConfigAgent {
 		errorMetrics: prometheus.NewCounterVec(prometheus.CounterOpts{}, []string{"error"}),
 	}
 	a.reloadConfig = func() error {
-		indexes := a.buildIndexes(a.configs)
 		a.lock.Lock()
-		a.indexes = indexes
```

Comment:
This confused me, why do we have both `loadFilenameToConfig` and `reloadConfig`?

Reply:
It looks like it is used to implement the fake version of the agent (see `NewFakeConfigAgent` vs. `NewConfigAgent`).